build/hpc_install.sh

```diff
@@ -0,0 +1,157 @@
+    git clone https://github.com/HopkinsIDD/flepiMoP.git $FLEPI_PATH
+elif [ ! -d "$FLEPI_PATH/.git" ]; then
+    echo "The flepiMoP found at '$FLEPI_PATH' is not a git clone, unsure of how to proceed."
+    exit 1
+fi
+
+# Setup the conda environment
+if [ ! -d "$USERDIR/flepimop-env" ]; then
+cat << EOF > $USERDIR/environment.yml
+channels:
+- conda-forge
+- defaults
+dependencies:
+- python=3.10
+- pip
+- r-base>=4.4
+- r-essentials
+- r-devtools
+- pyarrow=17.0.0
+- r-arrow=17.0.0
+# Manually specify this one because of the paths for libudunits2 on longleaf
+- r-sf 
+# This packages are probably missing from the DESCRIPTION of the R packages
+- r-optparse
+- pip:
+    - git+https://github.com/HopkinsIDD/flepiMoP.git#subdirectory=flepimop/gempyor_pkg
+EOF
+conda env create --prefix $USERDIR/flepimop-env --file $USERDIR/environment.yml
+fi
+conda activate $USERDIR/flepimop-env
+conda update --all
+
+# Check the conda environment is valid
+WHICH_PYTHON=$( which python )
+WHICH_R=$( which R )
+WHICH_PYTHON_OKAY=$( echo "$WHICH_PYTHON" | grep "flepimop-env" | wc -l )
+WHICH_R_OKAY=$( echo "$WHICH_R" | grep "flepimop-env" | wc -l )
+if [[ "$WHICH_PYTHON_OKAY" -ne 1 ]]; then
+    echo "The python found is '$WHICH_PYTHON', which does not contain the expected 'flepimop-env'."
+    exit 1
+fi
+if [[ "$WHICH_R_OKAY" -ne 1 ]]; then
+    echo "The R found is '$WHICH_R', which does not contain the expected 'flepimop-env'."
+    exit 1
+fi
+PYTHON_ARROW_VERSION=$( python -c "import pyarrow; print(pyarrow.__version__)" )
+R_ARROW_VERSION=$( Rscript -e "cat(as.character(packageVersion('arrow')))" )
+COMPATIBLE_ARROW_VERSION=$( echo "$R_ARROW_VERSION" | grep "$PYTHON_ARROW_VERSION" | wc -l )
+if [[ "$COMPATIBLE_ARROW_VERSION" -ne 1 ]]; then
+    echo "The R version of arrow is '$R_ARROW_VERSION' and the python version is '$PYTHON_ARROW_VERSION'. These may not be compatible versions."
+fi
+
+# Install the local R packages
+INSTALL_R=$( mktemp )
+cat << EOF > $INSTALL_R
+library(devtools)
+devtools::install_github('HopkinsIDD/flepiMoP', subdir='flepimop/R_packages/flepicommon')
+devtools::install_github('HopkinsIDD/flepiMoP', subdir='flepimop/R_packages/flepiconfig')
+devtools::install_github('HopkinsIDD/flepiMoP', subdir='flepimop/R_packages/inference')
+EOF
+Rscript $INSTALL_R
+rm $INSTALL_R
+
+# Set correct env vars
+export FLEPI_STOCHASTIC_RUN=false
+export FLEPI_RESET_CHIMERICS=TRUE
+export TODAY=`date --rfc-3339='date'`
+
+echo -n "Please set a project path (relative to '$USERDIR'): "
+read PROJECT_PATH
+export PROJECT_PATH="$USERDIR/$PROJECT_PATH"
+
+echo -n "Please set a config path (relative to '$PROJECT_PATH'): "
+read CONFIG_PATH
+export CONFIG_PATH="$PROJECT_PATH/$CONFIG_PATH"
+
+echo -n "Please set a validation date (today is $TODAY): "
+read VALIDATION_DATE
+
+echo -n "Please set a resume location: "
+read RESUME_LOCATION
+
+echo -n "Please set a flepi run index: "
+read FLEPI_RUN_INDEX
```

Comment:
per @jcblemai discussion: this is a run-specific set of variables, rather than flepimop install level variables.

Reply:
Now handled by `build/flepi_init.sh`.